src/foundation.cs

```diff
@@ -8852,57 +8899,67 @@ interface NSObject2 : NSObjectProtocol {
 		[Export ("exposedBindings")]
 		NSString[] ExposedBindings ();
 #else
+		[NoiOS][NoMacCatalyst][NoWatch][NoTV]
```

Comment:
I think you have to add these attributes in the  `#if !NET` branch above, otherwise we'll add those members to legacy Xamarin.

Reply:
Good catch.